src/AcceptanceTestsHolder/App_Packages/NSB.AcceptanceTests/NServiceBusAcceptanceTest.cs

```diff
@@ -15,6 +15,7 @@ public abstract partial class NServiceBusAcceptanceTest
         [SetUp]
         public void SetUp()
         {
+            System.Configuration.ConfigurationManager.GetSection("MySQL");
```

Comment:
I'm assuming there must be a fairly ridiculous story to go along with this line of code.